airflow/models/baseoperator.py

```diff
@@ -719,6 +724,8 @@ class derived from this one results in the creation of a task object,
         "on_skipped_callback",
         "do_xcom_push",
         "multiple_outputs",
+        # TODO: should executor go here? I think so
+        "executor",
```

Comment:
Looking for feedback on this, thanks!

Reply:
I said this before, the entire dict is pretty sketchy.

Reply:
If in doubt, mimic what you have there. _comps is used to calculate hash. if you do not add it here, then two identical tasks with different executor might have same hash. It's certainly wrong no matter what hash is used for. So yes it should be added here.

Reply:
> If in doubt, mimic what you have there. _comps is used to calculate hash. if you do not add it here, then two identical tasks with different executor might have same hash. It's certainly wrong no matter what hash is used for. So yes it should be added here.

Yupp, this was my understanding as well. Thanks for the verification!